cmd/cli/command/access/ensure.go

```diff
@@ -79,7 +83,16 @@ var ensureCommand = cli.Command{
 						Lookup: roles[i],
 					},
 				},
-			})
+			}
+
+			if duration != nil && duration[i] != "" {
```

Comment:
This would panic if you don't supply the right amount of durations because the length of durations slice is not checked above